src/devtools/views/components/ComponentInspector.vue

```diff
@@ -74,7 +74,23 @@ export default {
       } else {
         window.alert('DOM inspection is not supported in this shell.')
       }
+    },
+    onTitleClick () {
+      const file = this.target.file
+      if (file) {
+        fetch(`/_open?file=${file}`).then(() => {
```

Comment:
So this will be making a request from the origin of the extension and only works during dev - however we should be able to initiate the request from the inspected window by doing an eval like in `inspectDOM`!